src/client.h

```diff
@@ -378,9 +409,14 @@ protected slots:
     void OnJittBufSizeChanged ( int iNewJitBufSize );
     void OnReqChanInfo() { Channel.SetRemoteInfo ( ChannelInfo ); }
     void OnNewConnection();
-    void OnCLDisconnection ( CHostAddress InetAddr ) { if ( InetAddr == Channel.GetAddress() ) { emit Disconnected(); } }
-    void OnCLPingReceived ( CHostAddress InetAddr,
-                            int          iMs );
+    void OnCLDisconnection ( CHostAddress InetAddr )
```

Comment:
This one arguably should be multi-line, due to the `if`.  But that means it should have a blank line before and after it, too.

Reply:
apart from the optional "include grouping", clang-format does not add nor remove blank lines, if needed they need to be added manually

Reply:
Not sure how we should capture this to fix up afterwards.